.github/workflows/pythonpackage.yml

```diff
@@ -28,7 +29,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macOS-latest, windows-2016]
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.6, 3.7, 3.8, 3.9, 3.10.0-beta.1]
```

Comment:
Thanks! I think you need `3.10-dev` here - GH actions won't understand that.

Reply:
> looks like we may need to update the python actions to v2 as well

Ah, didn't realize that, thanks! Though currently some tests are failing for all versions, something is not correct yet...

Reply:
>  I'll try to take a peak at the failures in later today/tomorrow

Thanks - I can also have another look if you want, but probably not before tomorrow evening (noon for you).